apps/users/tests/test_forms.py

```diff
@@ -179,6 +179,16 @@ def test_no_names(self):
         r = self.client.post(self.url, data)
         self.assertFormError(r, 'form', 'username', 'This field is required.')
 
+    def test_fxa_id(self):
```

Comment:
could you change this test name to something more specific or add a comment explaining what it's testing for?